.github/workflows/action.yml

```diff
@@ -125,7 +125,6 @@ jobs:
         env:
           DB_PASSWORD: postgres
           DB_USERNAME: postgres
```

Comment:
question: Why is this deleted?

Reply:
as discussed `split-tests` doesn't exist as a step anywhere